great_expectations/checkpoint/actions.py

```diff
@@ -48,6 +48,10 @@ class ValidationAction:
     def __init__(self, data_context) -> None:
         self.data_context = data_context
 
+    @property
+    def _cloud_mode(self) -> bool:
+        return hasattr(self.data_context, "cloud_mode") and self.data_context.cloud_mode
```

Comment:
We'll want to clean this up eventually but Cloud leaks into everything so this is one way to encapsulate it a bit

Reply:
Why not just keep the existing `self.data_context.cloud_mode` until we can improve the abstraction?
This change seems lateral.

Reply:
not all contexts have cloud_mode